src/views/Playground.vue

```diff
@@ -230,7 +246,7 @@ onMounted(async () => {
       :open="modalNetworksOpen"
       @close="modalNetworksOpen = false"
       v-model="form.network"
-      @update:modelValue="handleURLUpdate"
+      @update:modelValue="event => handleURLUpdate(event, 'networkUpdate')"
```

Comment:
Why do we need to pass `event` into `handleURLUpdate()`?

Reply:
@samuveth, I made it so that the first parameter will always be the input event. I've used handleURLUpdate() in other places too (eg: https://github.com/snapshot-labs/snapshot/blob/1ba8bd1b93b6779240d4c8e73d8180c8ac969e82/src/views/Playground.vue#L168) and here event is passed as the first param implicitly.

The event param can be removed though